docs/choreolib/auto-routines.md

```diff
@@ -196,14 +196,7 @@ public Command fivePieceAutoTriggerMono(AutoFactory factory) {
   // shoots the note when the shoot event marker is reached
   trajectory.atTime("shoot").onTrue(shootIfNoteOwned());
 
-  // aims the shooter when the aim event marker is reached,
-  // the aim command aims based on the next shoot event marker position
-  final AtomicInteger shootIndex = new AtomicInteger(0);
```

Comment:
Should maybe change from `onTrue(shootIfNoteOwned());` to like `onTrue(aim().andThen(shootIfNoteOwned()));`